File: Codewars/Python/7 kyu/Arithmetic progression.py
def arithmetic_sequence_elements(a, r, n):
    resp=[]
    resp.append(str(a))
    while n>1:
        a+=r
        resp.append(str(a))
        n-=1
    return ", ".join(resp)

def arithmetic_sequence_elements_up(a, r, n):
    return ', '.join(str(a + b * r) for b in range(n))

print(arithmetic_sequence_elements(1, 2, 5))
print(arithmetic_sequence_elements_up(1, 2, 5))

File: Codewars/Javascript/7 kyu/III/Arithmetic progression.js
function arithmeticSequenceElements(a,r,n) {
    for(var i=0, resp=[a]; i<n-1; i++)
        resp.push(a+=r)
    return resp.join(', ')
}

const arithmeticSequenceElementsUp = (a,r,n) => Array.from({length: n}, (_,i) => a+r*i).join(', ')

const arithmeticSequenceElementsUp2 = (a,r,n) => Array(n).fill().map((_,k) => a + k*r).join(', ')

console.log(arithmeticSequenceElements(1, 2, 5));
console.log(arithmeticSequenceElementsUp(1, 2, 5));
console.log(arithmeticSequenceElementsUp2(1, 2, 5));